__tests__/app.test.js

```diff
@@ -34,3 +34,22 @@ describe("api/topics", () => {
       });
   });
 });
+
+describe.only("/api", () => {
+  test("should return an object describing all available endpoints", () => {
+    return request(app)
+      .get("/api")
+      .then(({ body }) => {
+        const sampleEndPoint = {
+          description:
+            "serves up a json representation of all the available endpoints of the api",
+          queries: [],
+          exampleResponse: {
+            topics: "/api/topics",
+            articles: "/api/articles",
+          },
```

Comment:
You can require the JSON in here instead of writing out the expected object. 

This should help as the JSON gets bigger later.